mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -2113,6 +2169,20 @@ llvm::SetVector<int64_t> BroadcastOp::computeBroadcastedUnitDims() {
                                       getResultVectorType().getShape());
 }
 
+llvm::SetVector<int64_t> BroadcastOp::computeRankExtendedDims() {
```

Comment:
[nit] Could you add a comment? I guess there are two types of broadcasts: 
* "rank-extending"
* "dim-1"
? Just double checking that my understanding is correct - I could find this documented anywhere.